by-language/python-sqlalchemy/README.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight: console
+
+############################################################
+Connect to CrateDB and CrateDB Cloud using Python/SQLAlchemy
+############################################################
+
+
+*****
+About
+*****
+
+Example programs demonstrating CrateDB's SQLAlchemy adapter and dialect.
+
+
+*****
+Usage
+*****
+
+To start a CrateDB instance on your machine for evaluation purposes, invoke::
+
+    docker run -it --rm --publish=4200:4200 --publish=5432:5432 crate
```

Comment:
Is publishing the PG port `5432` required for this example?

Reply:
Erm. Probably not. It is just my canonical command I slap everywhere, not needing to discriminate between different variants, and also advertising a bit